crates/hir-ty/src/consteval/tests.rs

```diff
@@ -2713,12 +2713,11 @@ fn const_trait_assoc() {
         r#"
     //- minicore: size_of, fn
     //- /a/lib.rs crate:a
-    use core::mem::size_of;
     pub struct S<T>(T);
     impl<T> S<T> {
         pub const X: usize = {
             let k: T;
-            let f = || core::mem::size_of::<T>();
+            let f = || size_of::<T>();
```

Comment:
This is why CI fails. This either needs to be undone or the corresponding prelude changes need to be done to the minicore.rs file.

Reply:
I opted to add `size_of` to the prelude (and add `prelude::rust_2024`). Let me know if you'd rather I'd have kept these uses qualified.